scripts/test/CrystalFieldMultiSiteTest.py

```diff
@@ -1,5 +1,7 @@
 import numpy as np
 import unittest
+
+import mantid # noqa
```

Comment:
What is this import for?

Reply:
It now needs this so that `mantid` will add the scripts directory to the python path. Previously the [`testrunner`](https://github.com/mantidproject/mantid/pull/21152/files#diff-a2d6a01451724b0fc156cafdc10b3539R39) imported `mantid` by default but this causes and unnecessary slow down in tests that actually don't use `mantid` at all.